resources/views/admin/worklogs/edit.blade.php

```diff
@@ -89,4 +89,30 @@ class="form-control"
     <button type="submit" class="btn btn-primary">Save</button>
 </form>
 
+<h3>Feedbacks</h3>
+@if($worklog->feedbacks)
+    <div class="container">
+        <table class="table table-striped">
+            <thead>
+            <tr>
+                <th scope="col">Created At</th>
+                <th scope="col">Feedback</th>
+            </tr>
+            </thead>
+            <tbody>
+            @foreach($worklog->feedbacks as $feedback)
```

Comment:
is feedbacks eager loaded?

Reply:
its always better to use forElse rather that forEach. have alook over it
and its a good practise to add no data found if there is no data for tables insteead of using if else